app/views/clientsmodal.html

```diff
@@ -41,7 +41,9 @@ <h4 class="modal-title" id="addClientLabel">
       </div>
       <div class="form-group" ng-class="{ 'has-error' : ngError.cert }">
         <label><span>**</span> Certificate</label>
-        <textarea class="form-control" rows="5" ng-model="client.cert" ng-trim="false"></textarea>
+        <select class="form-control" style="width: auto;" ng-model="client.cert" ng-options="cert._id as cert.commonName for cert in trustedCerts">
+          <option value="">Select Certificate</option>
+        </select>
```

Comment:
Could we have this select only appear when secure is set to true and also instead of having select certificate as default could you change this to "Use default CAs". Also could you change the heading to "Host certificate authority". Also, could you display the commom name as follows: "cn=trust.org".